static/sass/_pattern_search-panel.scss

```diff
@@ -12,8 +12,8 @@
     }
 
     &__group {
-      padding-right: 1.5rem;
       max-width: 443px;
```

Comment:
please use rems not px - and wat is this value based on?

Reply:
based on the design, otherwise it'll expand to the entire width of the page, and you'll probably ask me to shorten than

Reply:
why not use columns on the parent? A hardcoded value is not going to behave well responsively.

Reply:
This is a `max-width`, it will not affect responsive, the `width` will be auto. But I can give it a try with columns. I think this will work better, because columns could also wrap the text

Reply:
@lyubomir-popov What is this search aligned with? From design it doesn't seem to be aligned with columns, at least not with any 25% split?

<img width="781" alt="image" src="https://github.com/canonical/canonical.com/assets/83575/70820f7a-721f-47bf-b47f-d91203000d49">

By the looks of it this is a standard 25/75 grid split, but search form seems to have some arbitrary width. Limiting it using columns will be quite arbitrary, because it would require some nested grid with something that looks like col-6 or col-7?